presidio-analyzer/presidio_analyzer/predefined_recognizers/__init__.py

```diff
@@ -32,8 +32,14 @@
 from .au_acn_recognizer import AuAcnRecognizer
 from .au_tfn_recognizer import AuTfnRecognizer
 from .au_medicare_recognizer import AuMedicareRecognizer
+from .in_Aadhaar_recognizer import InAadhaarRecognizer
+from .in_drivers_licence_recognizer import InDriverLicenceRecognizer
+from .blood_group_recognizer import BloodGroupRecognizer
+from .mac_address_recognizer import MACAddressRecognizer
+from .in_pan_recognizer import InPanRecognizer
+from .coordinate_recognizer import CoordinateRecognizer
+from .IN_LICENSE_PLATE_recognizer import InLicencePlateRecognizer
```

Comment:
The naming is not consistent,. why upper case in this one and not the others ? Let's maintain as is

Reply:
done